lib/resty/session/mysql.lua

```diff
@@ -88,6 +102,14 @@ local function exec(self, query)
   return ok, err
 end
 
+local function cleanup_check(storage)
+  if should_cleanup() then
+    local now = ngx.time()
+    exec(storage, fmt(CLEANUP, storage.table,      now))
+    exec(storage, fmt(CLEANUP, storage.table_meta, now))
```

Comment:
Can we do cleanup so that it does it on same roundtrip as it does SET/DELETE (aka concatenate it into `SQL` buffer)?

Also there are a lot of calls to `ngx.time()` can we pass it instead? The `:set` already gets `current_time` which we can pass forward. For `:delete`, I think we can add it (passing meta.timestamp perhaps)?

I think same applies to other storages.

Reply:
done thanks!